src/compositions/useCustomDefaultFlowRunsFilter.ts

```diff
@@ -1,8 +1,27 @@
 import { useLocalStorage } from '@prefecthq/vue-compositions'
-import { getCacheKey, FlowRunsFilter } from '..'
+import { ComputedRef, computed, Ref } from 'vue'
+import { getCacheKey, FlowRunsFilter, SavedSearchFilter, mapper } from '..'
 
 const customDefaultFlowRunsFilterKey = getCacheKey('prefect-ui-library-custom-default-flow-runs-filter')
 
-export function useCustomDefaultFlowRunsFilter(): ReturnType<typeof useLocalStorage<FlowRunsFilter | null>> {
-  return useLocalStorage<FlowRunsFilter>(customDefaultFlowRunsFilterKey)
+type UseCustomDefaultFlowRunsFilter = {
+  value: Ref<SavedSearchFilter | null>,
+  set: (value: SavedSearchFilter) => void,
+  remove: () => void,
+  asFlowRunsFilter: ComputedRef<FlowRunsFilter | null>,
+}
```

Comment:
This composition is super simple and I like it. But I found the naming a little confusing. I think I got thrown off by the composition being "FlowRunsFilter" but `value` being a `SavedSearchFilter`. 

The only suggestion I have would be to rename this to `useCustomDefaultSavedSearchFilter`.

Reply:
ah yeah. that makes sense. 

definitely was/still-getting-ahold-of the overarching data models here (savedsearch, savedFilters, flowrunsfilter, etc.) 
I think it _was_ the flow runs filter at one point hence the name and the awkward `asFlowRunsFilter` converter, but I realized i needed the whole SavedSearch. 

I think a source of confusion has been that the components are all for "SavedFilters" which seem to actually primarily deal w/ "SavedSearch" but it atm it seems coupled to flow runs specifically.

Reply:
I was realizing that myself when I was looking at this. Maybe a cleanup PR after this to unify the names of everything would be nice.